orm/src/test/java/org/hibernate/search/test/query/sorting/Explorer.java

```diff
@@ -50,9 +50,12 @@
 
 	@ManyToOne
 	@IndexedEmbedded
-	@Field(bridge = @FieldBridge(impl = Territory.IdFieldBridge.class))
-	@NumericField
-	@SortableField
+	/*
+	 * Don't store the id in "favoriteTerritory" directly as this could conflict
+	 * with the @IndexedEmbedded with the Elasticsearch backend
+	 */
+	@Field(name = "favoriteTerritory.idFromBridge", bridge = @FieldBridge(impl = Territory.IdFieldBridge.class))
```

Comment:
Interesting. In which case might we have a conflict? I suppose it's related to the null case (just a vague remembering of a previous PR)? Maybe we should add a prefix or a suffix for this field specific to Es to avoid conflicts?